debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnector.java

```diff
@@ -85,6 +85,10 @@ public Config validate(Map<String, String> connectorConfigs) {
         ConfigValue userValue = results.get(PostgresConnectorConfig.USER.name());
         ConfigValue passwordValue = results.get(PostgresConnectorConfig.PASSWORD.name());
 
+        if (passwordValue.value() == null || ((String)passwordValue.value()).isEmpty()) {
+            logger.warn("The connection password is empty, this is valid setting only for special cases");
```

Comment:
I'm a bit torn on that one. We shouldn't lecture people in our logs without good reason :) How about removing the second part of the sentence (after the comma)?